### DISCUSSION TITLE
Block brut force

### DISCUSSION THREAD

**Discussion by S1M8N** [hassio-addons/addon-bitwarden#171] [Q&A]
Hey, I have a question, I use home assistant OS and I have bitwarden. When I put incorrect passwords in my bitwarden I'm never blocked. Do you have solution how can I make "**login_attempts_threshold**" in my bitwarden when I use home assistant ? Thanks in advenced

**Comment by nicolasvila** [comment]
Maybe this might be something to be implemented in Vaultwarden code base? (not in HomeAssistant add-on)
https://github.com/dani-garcia/vaultwarden

**Reply by S1M8N** [reply]
Thank you for your answer, I will go check with them
